get_staples.hh

```diff
@@ -30,3 +30,43 @@ template<class T, class S> void get_staples(T &K, gaugeconfig<S> &U,
   }
 }
 
+//stores c1*U1+complex(c2*U2) in U1
+//variable K used for accumulation is of type Complex for u1 (and su2 for su2?)
+inline void scalarmultiplyadd(Complex &U1, _u1 U2, double c1, double c2) {
+    U1=Complex(c1,0.)*U1+Complex(c2,0.)*Complex(U2);
+}
+
+//stores c1*U1+c2*U2 in U1
+void scalarmultiplyadd(_su2 &U1, _su2 U2, double c1, double c2) {
```

Comment:
Could you please explain what is the purpose of ```scalarmultiplyadd()```? I understand what it does but now why it should be clearer or more efficient than calling:
```K += // expression ```

Reply:
I wrote it when I was new to C++ style syntax and unsure about operator overload and the structure accum, it should not be in use anymore.